src/box/memtx_engine.cc

```diff
@@ -1327,6 +1311,11 @@ memtx_join_f(va_list ap)
 		while (true) {
 			RegionGuard region_guard(&fiber()->gc);
 			struct read_view_tuple result;
+			if (fiber_is_cancelled()) {
+				diag_set(FiberIsCancelled);
+				rc = -1;
+				break;
+			}
```

Comment:
Again, looks redundant because `memtx_join_send_tuple` calls `relay_initial_join` via `xstream_write`, which is a cancellation point.

Reply:
It is not yet a cancellation point. But after discussion f2f we decided to make `coio_write_timeout` a cancellation point. So this change is dropped.